### ISSUE TITLE
Cannot find certain rules

### ISSUE THREAD

**Issue by ricenzca** [mgechev/revive#621]
**Describe the bug**
Cannot find the following rules:
- time-equal
- banned-characters
- optimize-operands-order

**To Reproduce**
1. I installed revive `go install github.com/mgechev/revive@latest`
2. I run it with the following flags & configuration file:

```shell
revive -formatter friendly -config default.toml ./...
```

```toml
# When set to false, ignores files with "GENERATED" header, similar to golint
ignoreGeneratedHeader = true

# Sets the default severity to "warning"
severity = "warning"

# Sets the default failure confidence. This means that linting errors
# with less than 0.8 confidence will be ignored.
confidence = 0.8

# Sets the error code for failures with severity "error"
errorCode = 1

# Sets the error code for failures with severity "warning"
warningCode = 1

[rule.time-equal]
[rule.banned-characters]
[rule.optimize-operands-order]
```

**Expected behavior**
I expect the rule to be present and executed

**Logs**
![image](https://user-images.githubusercontent.com/48046395/149486497-ab4dc333-5960-4bbb-8560-b34f2ca17b3c.png)

**Desktop (please complete the following information):**
 - OS: macOS Big Sur
 - Version of Go: 1.16.8

**Additional context**
NA

**Comment by chavacava** [comment]
Hi @ricenzca, thanks for filling the issue.
I was unable to reproduce the problem.

```bash
docker run -it golang:1.16-alpine sh -c 'go install github.com/mgechev/revive@master && revive -version && /bin/sh'
Unable to find image 'golang:1.16-alpine' locally
1.16-alpine: Pulling from library/golang
59bf1c3509f3: Pull complete 
666ba61612fd: Pull complete 
8ed8ca486205: Pull complete 
fa6e261410ce: Pull complete 
033ff68f5bd2: Pull complete 
Digest: sha256:c4b36cff558405c8368be606325261a21a9a3ae9f79dc1bd3fff9f831a0411f2
Status: Downloaded newer image for golang:1.16-alpine
go: downloading github.com/mgechev/revive v1.1.3-0.20220101011118-af953e61891d
go: downloading github.com/fatih/color v1.13.0
go: downloading github.com/mgechev/dots v0.0.0-20210922191527-e955255bf517
go: downloading github.com/mitchellh/go-homedir v1.1.0
go: downloading github.com/mattn/go-colorable v0.1.9
go: downloading github.com/mattn/go-isatty v0.0.14
go: downloading github.com/BurntSushi/toml v0.4.1
go: downloading golang.org/x/tools v0.1.8
go: downloading golang.org/x/sys v0.0.0-20211019181941-9d821ace8654
go: downloading github.com/chavacava/garif v0.0.0-20210405164556-e8a0a408d6af
go: downloading github.com/olekukonko/tablewriter v0.0.5
go: downloading github.com/fatih/structtag v1.2.0
go: downloading github.com/mattn/go-runewidth v0.0.9
version 1.1.3-0.20220101011118-af953e61891d
/go # go version
\go version go1.16.13 linux/amd64
/go # cd bin/
/go/bin # ./revive -version
version 1.1.3-0.20220101011118-af953e61891d
/go/bin # ./revive ./...
/go/bin # vi default.toml
/go/bin # cat default.toml 
# When set to false, ignores files with "GENERATED" header, similar to golint
ignoreGeneratedHeader = true

# Sets the default severity to "warning"
severity = "warning"

# Sets the default failure confidence. This means that linting errors
# with less than 0.8 confidence will be ignored.
confidence = 0.8

# Sets the error code for failures with severity "error"
errorCode = 1

# Sets the error code for failures with severity "warning"
warningCode = 1

[rule.time-equal]
[rule.banned-characters]
[rule.optimize-operands-order]

/go/bin # ./revive -formatter friendly -config default.toml ./...
/go/bin # 
```

**Comment by chavacava** [comment]
Hi @ricenzca , could you please check if the `revive` binary you are executing is the very one you've installed with `go install github.com/mgechev/revive@latest`?

Does `$ which revive` result in the `revive` binary you expect?

thanks

**Comment by chavacava** [comment]
No feedback from @ricenzca, I'll close the issue. We can re-open it if necessary.
